packages/patternfly-4/react-styled-system/src/components/StyledSystem/StyledTheme.d.ts

```diff
@@ -161,6 +162,7 @@ export interface StyledConstants {
 }
 
 export interface PatternFlyThemeProviderProps extends HTMLProps<HTMLDivElement> {
+  // eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
Do you know why we need the `Object` type here? If we can safely remove the general type in favor of the primitive `object` I think that's the way to go, then we don't need the ignore eslint statement

Reply:
Updated based on [https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#general-types](feedback)